### PR TITLE
Blob: Explosive Lattice now actually works, reactive spines retaliation applies reagent, description consistency

### PR DESCRIPTION
**Pull request by NotRanged**
## About The Pull Request

The Explosive Lattice strain was _completely_ broken. In the images below, the middle mob is attacked once.
image

It doesn't do an AoE at all, does 5 dmg from the default blob_act on the target, and gives a single guy in the 'AoE' 15 damage.

Explosive Lattice has been changed to use the same damage calculations as ex_act, dealing a mix of brute/burn in the AoE properly.

image

Additionally, the Reactive Spines retaliation now properly smacks living mobs with the reagents that the strain should apply. A small 2.5 second cooldown has been added between these retaliatory attacks, considering the increased damage they do to mobs now.

The strain effect descriptions were also _all over the place_ in terms of damage applied, and have been made more consistent. E.g. an 0.6 reagent mod was called 'high oxy damage' while a 1.0 mod was called 'medium brute' elsewhere.

## Why It's Good For The Game

Blob code is an inconsistent fucking mess that I've been slowly working away at making more bearable, this is another step forward (I hope). Broken strain fixed, another strain's behavior made more sensible, and various comment/description fixes.

## Changelog
:cl:
add: The Reactive Spines blob strain's retaliation now applies the strain's reagents, and can trigger once every 2.5 seconds.
fix: Fixed and changed the Explosive Lattice blob strain. It didn't work at all, but now actually applies an AoE of light burn/brute.
code: Changed comments and descriptions of various blob strains to be more consistent.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
I want to say some of that inconsistency is from various code changes over time.

By the way, do you happen to know why currently the reactive spines aren't doing the attack animation when they react?

**Comment by NotRanged** [comment]
I'm not sure how the blob_attack_animation() would look when it attacks multiple things at once, like it does in the AoE retaliate. I'll check it if looks good when added.

File: code/modules/antagonists/blob/blobstrains/explosive_lattice.dm [hunk 1]
```diff
@@ -33,22 +33,42 @@
...
-			if(ROLE_BLOB in nearby_mob.faction) //no friendly fire
+
+		// Total damage to epicenter mob of 0.7*reac_volume, like a mid-tier strain
+		brute_loss = reac_volume*0.35
+		burn_loss = brute_loss
+
+		bomb_armor = exposed_mob.getarmor(null, BOMB)
+		if(bomb_armor) // Same calculation and proc that ex_act uses on mobs
+			brute_loss = brute_loss*(2 - round(bomb_armor*0.01, 0.05))
+			burn_loss = brute_loss
```
Comment (Rohesie) [inline code/modules/antagonists/blob/blobstrains/explosive_lattice.dm:53]:
```suggestion

		bomb_armor = exposed_mob.getarmor(null, BOMB)
		if(bomb_armor) // Same calculation and proc that ex_act uses on mobs
			brute_loss = brute_loss*(2 - round(bomb_armor*0.01, 0.05))

		burn_loss = brute_loss
```

**Comment by Rohesie** [comment]
Tested locally and working?

**Comment by NotRanged** [comment]
Yep

**Pull request merged**